softhsm/Dockerfile

```diff
@@ -1,27 +1,33 @@
-from alpine:latest as build
-workdir /softhsm
+from openjdk:11-jre-slim as build
+workdir /work
 
 # Build softHSM source
 arg version=2.5.0
-run apk --update add opensc build-base wget sqlite-dev openssl-dev
+
+run apt-get update && \
+    apt-get install -y \
+    opensc build-essential wget libsqlite3-dev libssl-dev \
+    cmake libseccomp-dev libengine-gost-openssl1.1
+
 run wget https://dist.opendnssec.org/source/softhsm-${version}.tar.gz && \
     tar xvf softhsm-${version}.tar.gz && \
     cd softhsm-${version} && \
     ./configure && \
     make && make install
 
 # Build pkcs11-proxy
-run apk --update add cmake bash linux-headers libseccomp-dev
 run wget https://github.com/SUNET/pkcs11-proxy/archive/master.zip && \
     unzip master.zip && \
     cd pkcs11-proxy-master && \
     cmake . && make && make install
 
 # Run image
-from alpine:latest
-workdir /softhsm
+from openjdk:11-jre-slim
+workdir /work
```

Comment:
We should probably have a more descriptive name for the application director

Reply:
I'm going to stick with the `/app` convention we have on the Java `Dockerfile`s, if that's ok.